pkgs/by-name/go/goshs/package.nix

```
@@ -0,0 +1,32 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "goshs";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "patrickhener";
+    repo = "goshs";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
See https://github.com/NixOS/nixpkgs/pull/355973